web/app/services/config.ts

```diff
@@ -12,6 +12,7 @@ export default class ConfigService extends Service {
     feature_flags: config.featureFlags,
     google_doc_folders: config.google.docFolders ?? "",
     short_link_base_url: config.shortLinkBaseURL,
+    google_analytics_tag_id: config.googleAnalyticsTagId
```

Comment:
Hmm, does this work? It looks like `config.googleAnalyticsTagId` was removed in this PR [here](https://github.com/hashicorp-forge/hermes/pull/143/files#diff-5d61833860ee05d29b82a217c3ad8a57ac5ce86bf8e07e486b94bad91b37ec8cL14)?

Reply:
That line was just the check for whether the variable existed, from when the config was simpler.

Now [we pass in that variable regardless of whether it exists](https://github.com/hashicorp-forge/hermes/pull/143/files#diff-5d61833860ee05d29b82a217c3ad8a57ac5ce86bf8e07e486b94bad91b37ec8cR40). Previously, that would throw errors; but now we have code to [deal with that scenario](https://github.com/hashicorp-forge/hermes/pull/143/files#diff-0eb08ae4ac297fb24e5c9a69be04f4197442d01e06d7b822b325bcc673923fd3R19-R24).

Reply:
I wonder if we wouldn't need to override that class if this line was something like:
```
google_analytics_tag_id: config.metricsAdapters[0].config.id
```
(there's probably a way to make this less brittle than just referencing the first element too?)